File: app-service/src/main/java/com/devproblems/controller/IndexController.java
package com.devproblems.controller;

import com.go.starter.service.CoderAcademyService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/")
public class IndexController {

    @Autowired
    private CoderAcademyService coderAcademyService;


    @GetMapping("one")
    public String one() {
        return "success -> " + coderAcademyService.sayHello();
    }
}

File: app-service/src/main/java/com/go/controller/IndexController.kt
package com.go.controller

import com.alibaba.fastjson2.JSON
import com.go.starter.service.IProcessService
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/")
class IndexController(
    private val processService: IProcessService
) {
    @GetMapping("insert")
    fun one(): String {
        processService.deploy("xxxxxxxxxx")
        val sss = processService.detailByProcessKeyForce("process_key_001")
        return "success -> ${JSON.toJSONString(sss)}"
    }
}